cmd/ndm_daemonset/probe/uuid.go

```diff
@@ -53,6 +54,14 @@ func generateUUID(bd blockdevice.BlockDevice) (string, bool) {
 	// the partition and the unique data will be stored in sectors where consumers do not have access.
 
 	switch {
+	/*case bd.DeviceAttributes.DeviceType == blockdevice.BlockDeviceTypeLoop:
```

Comment:
Is this because we haven't added the ability to mark a block device as loop device in udev package yet?

Reply:
Its because we havent decided how we should generate UUID for loop devices. Since loop devices are only present virtually on the node, we may use the `devicename + hostname` method here.

cc: @kmova need your thoughts on this.

Reply:
Will the loop names change on reboots @akhilerm. 

Can we read a few chars from the loop device and then use them to generate an ID?

Reply:
> Will the loop names change on reboots @akhilerm.

No @kmova , it wont.

> Can we read a few chars from the loop device and then use them to generate an ID?

That means, the backing file right? Its content can change.